src/main/resources/db/changelog/changes/06-ensure-admin-user.yml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: ensure-admin-user
+      author: Anton
+      preConditions:
+        - and:
+          - sqlCheck:
+              expectedResult: 0
+              sql: SELECT COUNT(*) FROM users u JOIN users_roles ur ON u.id = ur.user_id JOIN roles r ON ur.role_id = r.id WHERE r.role_name = 'ROLE_ADMIN'
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "admin@email.com"
+              - column:
+                  name: password
+                  value: "$2a$10$gIV9deZb4E80By2j0F3fwOtkcRFRgE8/AIjEbLDKpTUg8uTWGqh9C"
```

Comment:
add a password in plaintext as a comment here, because it's useless for testing otherwise